config/owasp/suppressions.xml

```diff
@@ -23,6 +23,15 @@
     </notes>
     <cve>CVE-2022-29885</cve>
   </suppress>
+  
```

Comment:
@dinesh1patel can you please move all the temporary suppression to one element?

Reply:
@rajaram-nagarajan done